cysimdjson/cysimdjson.pyx

```diff
@@ -152,6 +175,21 @@ cdef class JSONElement:
 		return ok == 0
 
 
+	def __iter__(self):
+
+		for _key in self.keys():
+			yield _key
+
+
+	def items(self):
+
+		cdef simdjson_element v
+
+		for _key in self.keys():
```

Comment:
And again ... we should use an C++ iterator, if there is a one on SIMDJSON level.